.github/renovate.json5

```diff
@@ -60,6 +60,18 @@
         "v1.26",
       ]
     },
+    {
```

Comment:
i'd assume that we need an explicit regexMatcher that matches the image in the `ARG`.

By default [Renovate's language support for Docker](https://docs.renovatebot.com/docker/#how-it-works) only searches in `FROM` directives by default.